config/development/recreationalGames.js

```diff
@@ -7,37 +7,9 @@
         Stellaris: {
```

Comment:
Upon reviewing this in the diff I think it would be far more sensible to keep multiple recreational game objects for testing purposes